cernopendata/modules/fixtures/data/docs/atlas-open-data-for-research-release-2024/atlas-open-data-for-research-release-2024.md

```diff
@@ -0,0 +1,34 @@
+## Explore over 7 billion LHC collision events – from home
+
+The ATLAS Experiment at CERN has made two years’ worth of scientific data available to the public for research purposes. The data include recordings of proton–proton collisions from the Large Hadron Collider (LHC) at a collision energy of 13 TeV. This is the first time that ATLAS has released data on this scale, and it marks a significant milestone in terms of public access and utilisation of LHC data.
+
+“Open access is a core value of CERN and the ATLAS Collaboration,” says Andreas Hoecker, ATLAS Spokesperson. “Since its beginning, ATLAS has strived to make its results fully accessible and reusable through open access archives such as [arXiv](https://arxiv.org/search/?query=ATLAS&searchtype=author&source=header) and [HepData](https://www.hepdata.net/search/?q=&collaboration=ATLAS). ATLAS has routinely released open data for educational purposes. Now, we’re taking it one step further — inviting everyone to explore the data that led to our discoveries.”
+
+Released under the [Creative Commons CC0 waiver](https://www.google.com/url?q=https://creativecommons.org/publicdomain/zero/1.0/legalcode.en&sa=D&source=docs&ust=1718970196670398&usg=AOvVaw1PABiSus03t0AxC3TYH5RL), ATLAS has made public all the data collected by the experiment during the 2015 and 2016 proton–proton operation of the LHC. This is approximately 65 TB of data, representing over 7 billion LHC collision events. In addition, ATLAS has released 2 billion events of simulated “Monte Carlo” data, which are essential for carrying out a physics analysis.
```

Comment:
The link looks weird. Could you please confirm that it is correct?

Reply:
Good point! Should of course just be https://creativecommons.org/publicdomain/zero/1.0/legalcode.en

Reply:
I'll make a fix, together with making a local copy of the external illustrating image, so that we don't have to change the image-loading configuration on the production application.